### ISSUE TITLE
Can't interact with confirmation dialog pop-up

### ISSUE THREAD

**Issue by cohnhead66** [seleniumbase/SeleniumBase#2514]
Thanks to your help I was able to make a lot of progress.  I have been able to get to the final automation step without triggering Captcha.  

Last steps are:

- Click "Submit" button
- Click "Confirm" pop-up
- Click "Exit"
- Driver quit.

Is is when I click submit button using uc_click() the pop-up flashes briefly and then disappears, and Captcha page is displayed. and the data to be submitted is not saved.
when I a driver.reconnect("breakpoint") before the uc_click() and then click the "Submit" button manually I get the same result, (Flash of the dialog box, Captcha, and no saved data)

Any guidance appriciated

**Comment by mdmintz** [comment]
For handling the issue you see with pop-ups, see https://github.com/seleniumbase/SeleniumBase/issues/1825 and https://github.com/seleniumbase/SeleniumBase/issues/600#issuecomment-647270426.

**Comment by cohnhead66** [comment]
Thanks Michael, After following your guidance on the work around for the pop up, I still was facing the same issue with where as soon as it was displayed it triggered a Captcha.  It happened both when under test or if I disconnected with reconnect)"breakpoint".

Since I was having no problem with Captcha when performing the task manually, i thought it might be related to the proxy.  I disabled the proxy, and I triggered the Captcha much earlier in the script, but once I got past that, the script completed to the end, even the pop up was handled correctly.  Captcha also triggered at the same point if i disconnected driver and performed the same steps manually.

So i'm thinking maybe i should try a different proxy service.  Ultimate goal is to run these scripts on a VM in the cloud, so almost certain I will need a good working proxy.  Any thoughts on how else I can avoid the captcha when automating browser spun up by SeleniumBase?
